src/server_manager/web_app/data_formatting.ts

```diff
@@ -75,7 +78,11 @@ export function formatBytesParts(numBytes: number, language: string): DataAmount
     throw new Error('formatBytesParts only works in web app code. Node usage isn\'t supported.');
   }
   const params = getDataFormattingParams(numBytes);
-  const parts = makeDataAmountFormatter(language, params).formatToParts(params.value);
+  const formatter = makeDataAmountFormatter(language, params);
+  if (!formatter) {
```

Comment:
Same here. We should just assume the language is present, otherwise we will be hiding a bug. In the callers that need you can check the language before calling this (as we already do in outline-server-view)

Reply:
Reverted